bin/report.dart

```diff
@@ -1,10 +1,14 @@
+import 'dart:convert';
 import 'dart:io';
 
 import 'package:args/command_runner.dart';
 import 'package:flutter_github_scripts/github_datatypes.dart';
 import 'package:graphql/client.dart';
+import 'package:http/http.dart' as http;
 import 'package:path/path.dart' as path;
 
+String? get githubToken => Platform.environment['GITHUB_TOKEN'];
```

Comment:
Should this exit if no GITHUB_TOKEN is found in the env file?

Reply:
There's a check on the first use, on or around line 37 of this file.

Reply:
Oof, Sorry, you are correct here!